### PR TITLE
{bio}[intel/2017b] BamTools v2.5.0

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~~https://github.com/easybuilders/easybuild-easyblocks/pull/1332~~

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
gligar01.gligar.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/e98f2cce2e1578e2c4f8d0cfa3df520f for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2663.swalot.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/47a0cdaf380b47c889b1398c04953062 for a full test report.

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
rescomp2 - Linux centos 6.8, Intel(R) Xeon(R) CPU E5-2650 v2 @ 2.60GHz, Python 2.7.11
See https://gist.github.com/274d3255c3b453399441f51701c04868 for a full test report.

File: easybuild/easyconfigs/b/BamTools/BamTools-2.5.0-intel-2017b.eb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name = 'BamTools'
+version = '2.5.0'
+
+homepage = 'https://github.com/pezmaster31/bamtools'
+description = """BamTools provides both a programmer's API and an end-user's toolkit for handling BAM files."""
+
+toolchain = {'name': 'intel', 'version': '2017b'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/pezmaster31/bamtools/archive']
```
Comment (verdurin) [inline easybuild/easyconfigs/b/BamTools/BamTools-2.5.0-intel-2017b.eb]: @boegel - missing checksum...

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2074.delcatty.os - Linux centos linux 7.4.1708, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2465d89c33150102282d968ab55f76c7 for a full test report.

**Comment by verdurin** [comment]
Going in, thanks @boegel!

**Pull request merged**